src/Microsoft.ML.AutoML/Experiment/MetricsAgents/IMetricsAgent.cs

```diff
@@ -10,6 +10,6 @@ internal interface IMetricsAgent<T>
 
         bool IsModelPerfect(double score);
 
-        T EvaluateMetrics(IDataView data, string labelColumn);
+        T EvaluateMetrics(IDataView data, string labelColumn, string groupId = null);
```

Comment:
**Internal docs:** I think it would be useful to add a doc, or at least a simple comment, stating that the `groupId` parameter is only meant to be used by the `RankingMetricsAgent`.

Reply:
agreed!